src/components/menu/menu.tsx

```diff
@@ -20,6 +20,8 @@ import {
  * @exampleComponent limel-example-menu-basic
  * @exampleComponent limel-example-menu-disabled
  * @exampleComponent limel-example-menu-open-left
+ * @exampleComponent limel-example-menu-open-bottom
+ * @exampleComponent limel-example-menu-open-right-start
```

Comment:
I wonder if we need new examples? I think it's enough with one example about open direction, and we also have the composite example. So I'd suggest removing this, and instead writing some docs in the original example about possibility of opening the menu in 12 different locations (if you think we have to).